src/taco-utils/cordovaWrapper.ts

```diff
@@ -10,28 +10,34 @@
 /// <reference path="../typings/node.d.ts" />
 /// <reference path="../typings/Q.d.ts" />
 /// <reference path="../typings/semver.d.ts" />
+/// <reference path="../typings/tacoLiveReload.d.ts" />
+
 
 "use strict";
 
-import assert = require ("assert");
-import child_process = require ("child_process");
-import path = require ("path");
-import Q = require ("q");
-import semver = require ("semver");
-import util = require ("util");
+import assert = require("assert");
+import child_process = require("child_process");
+import path = require("path");
+import Q = require("q");
+import semver = require("semver");
+import util = require("util");
+import livereload = require("taco-livereload");
```

Comment:
You'll need to add `taco-livereload` to taco-utils package.json dependencies, otherwise this require will fail because it won't be able to find `taco-livereload`